src/js/fetching/fetch-movies.js

```diff
@@ -1,8 +1,9 @@
-import { setMovieGenresToStorage } from '../rendering/genres-rendering';
-import { renderMovies } from '../rendering/render';
-import { loadMovie } from '../utils/loader';
+import { saveMovieGenresToStorage } from '../rendering/render-genres';
+import { renderMovies } from '../rendering/render-movies';
+import { showLoader } from '../utils/loader';
+
+// ------> CONSTANTS USED IN THE PROJECT:
```

Comment:
The comment must be on the line above the content being described (no break line)